shapeAnalysisProject/shape_matching/hausdorff.py

```diff
@@ -0,0 +1,13 @@
+from scipy.spatial.distance import directed_hausdorff
+
+def similarity(new_image, images=[], similarity=[]):
+    print "Executing Phase 3: Calculating similarity of shapes using Hausdorff distance"
+    N = len(images)
+    for i in xrange(N):
+        similarity[i].append(max(directed_hausdorff(images[i], new_image)[0],directed_hausdorff(new_image,images[i])[0]))
```

Comment:
@NamrataGuptaRoy later on I will add CircleCI tests but as of now, please make sure that the length of a single line is not more than 79 characters.